### DISCUSSION TITLE
Should we deprecate Coffeescript package?

### DISCUSSION THREAD

**Discussion by harryadel** [meteor/meteor#13531] [Polls]
I noticed that Meteor still has a [dedicated page](https://docs.meteor.com/packages/coffeescript.html) for Coffeescript, along with a couple of mentions here and there. I think it's maybe time to deprecate it. What do you think?

**Comment by StorytellerCZ** [comment]
There will always be people who prefer Coffeescript and will swear by it, but I think it is pretty clear that at least for now TypeScript is the clear winner here.

**Reply by Torgen** [reply]
AIUI Coffeescript and Typescript share a problem, which is that you can't get coverage for dynamically-loaded client-side modules. It's why I decaffeinated to regular JS instead of TS.
